proposed/2023/vs-filters.md

```diff
@@ -0,0 +1,102 @@
+
+- "There are no filtering options"
+- "More search options would make this better"
+- "Would like to find packages by tags, authors, licenses, and more"
+- "There are no way to sort packages outside of creation date"
+- "I'm not aware of any filters but prerelease on/off"
+- "Often I'm looking for a widely-used and respected package and I have to filter and sort through random niche-market packages"
+- "I should be able to filter based on an official package source and license of a package"
+- "I manually have to use google or the console because desired results are too hard to find without filters"
+- "Filters and sorts do not match the website"
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or pseudocode to show how this proposal would look. -->
+
+When a developer is using Visual Studio to search and manage NuGet packages, they will see a new filter and sort experience. 
+
+The browse filter experience will allow the developer to filter results based on common concepts such as "frameworks", "prerelease", "owners", "download count", "license", and "created on". 
+
+![](resources/VSFilter/PackageOptions.png)
+
+The installed filter experience will allow the developer to filter results based on all the **browse filter** and additional package status filters such as "deprecated", "vulnerable", and "update".
+
+![](resources/VSFilter/PackageOptionsAndStatus.png)
+
+To apply filters, the developer will click the new filter button next to the search box. This will open up a new window of the possible filtering options to choose from.
+
+When a filter is selected, the filter will show up as a pill/badge affordance in the package list which can be removed by clicking on the "X" associated with the pill/badge. For some filters, they will support multiple values such as frameworks, owners, and licenses. For other filters, they will only support one value such as prerelease, download count, created on, etc. For filters that have popular values but also have custom values, a input box will be provided. For example the license filter will have popular OSS licenses, but with the ability to insert the name of a not well known license as well.
+
+![](resources/VSFilter/PillsAndSort.png)
+
+Whenever a filter is applied, it will additionally show up in the Visual Studio search box so that developers get used to the search prefix syntax and can apply filters more quickly in the future.
+
+The sorting experience will allow the developer to sort these options on common concepts such as "alphabetical", "version", "relevance", "downloads", and "recently updated".
```

Comment:
I think we're missing the clarity on what exact filters we want to do and the feasibility of them. 

We discussed some of those concerns in https://github.com/NuGet/Home/pull/12704/files#r1244465668, so it'd be great to incorporate that here. 

I think it might make it easier to review if there are different sections for installed vs updates.